src/librustdoc/html/render/mod.rs

```diff
@@ -843,7 +843,7 @@ themePicker.onblur = handleThemeButtonsBlur;
             &cx.shared.fs,
             cx.path("storage.js"),
             &format!(
-                "var resourcesSuffix = \"{}\";{}",
```

Comment:
We need to check if the quote is escaped then since it's a "user input".

Reply:
But the current one wasn't checked either, that could be a huge change aside from this patch. This one just unifies it to single quote where possible, some parts still have double quote because it is manually templating json.

Reply:
You're changing something working, so of course you have to expect things like this to be brought up. ;)

Reply:
I understand, but the previous version didn't unescape it too. Do you think I should do it in this patch?

Reply:
This is far from the only place that `resource_suffix` is used so fixing this should be done in a different PR.